main.go

```diff
@@ -214,6 +216,16 @@ Options:
 			*format = "[%T]"
 		}
 	}
+	if *delim == "" {
+		*delim = "\t"
```

Comment:
Hey @luckman212, thanks for the PR!

Unless there's a good reason for it, I would suggest to not change the default delimiter from space to tab though, for backward compatibility.

Apart from that, it looks good to me!

Reply:
Great, I just pushed another update and squashed the commits, so hopefully this one is good to go